docs/AccessScalarProducts.md

```diff
@@ -0,0 +1,178 @@
+# Expose ScalarDB or ScalarDL in a Kubernetes cluster environment
+
+This document explains how to expose ScalarDB or ScalarDL in a Kubernetes cluster environment. To expose ScalarDB or ScalarDL, you can use Scalar Envoy via a Kubernetes service resource named `<HELM_RELEASE_NAME>-envoy`. You can use `<HELM_RELEASE_NAME>-envoy` in several ways, such as:
```

Comment:
I feel if we say `expose`, it should be like `expose ScalarDB or ScalarDL endpoints/APIs`, but it still sounds a bit unnatural (indirect?) to me. Also, `in` makes things confusing since we actually expose endpoints outside the K8s cluster.

How about saying this?

```suggestion
This document explains how to make ScalarDB or ScalarDL deployed in a Kubernetes cluster environment available from applications. To expose ScalarDB or ScalarDL, you can use Scalar Envoy via a Kubernetes service resource named `<HELM_RELEASE_NAME>-envoy`. You can use `<HELM_RELEASE_NAME>-envoy` in several ways, such as:
```
@josh-wong What do you think?

Reply:
Thanks for the explanation. LGTM!

Reply:
Thank you for your suggestion!
It looks good to me. I applied it.